### DISCUSSION TITLE
Episode 200!

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#99] [Episode]
Episode Link: https://adspthepodcast.com/2024/09/20/Episode-200.html

Feel free to write comments or ask questions 🙂

**Comment by Spencer-Comin** [comment]
As someone a few years younger than you both, but also married and with kids, this was highly entertaining to me

**Comment by CharString** [comment]
If Bryce would listen to the episodes, he would have known that his hair isn't his most attractive feature, but his 10x squirrel voice. 🐿️ He may talk as if he's executive, but he puts the 10x in 10x developer.

**Reply by CharString** [reply]
PS maybe that's why he's so fond of walnut
